src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -634,9 +649,19 @@ export default class GeFormRenderer extends LightningElement{
             const formControls = this.getFormControls(event);
             formControls.toggleSpinner();
 
+            let tokenizedGift = null;
             try {
                 if (this.shouldTokenizeCard()) {
-                    await this.tokenizeCard(sectionsList)
+                    tokenizedGift = new ElevateTokenizeableGift(
+                        this.cardholderNames.firstName,
+                        this.cardholderNames.lastName,
+                        getCurrencyLowestCommonDenominator(
+                            this.getFieldValueFromFormState(DONATION_AMOUNT)
+                        ),
+                        CURRENCY,
```

Comment:
I think this can be imported directly internal to `ElevateTokenizeableGift`.

Reply:
You're right! Made that change. I think it makes the code look cleaner now that we're passing in less parameters to the TokenizeableGift constructor.